web/libs/editor/src/components/BottomBar/Controls.jsx

```diff
@@ -245,15 +245,17 @@ export const Controls = controlsInjector(
         }
 
         if ((userGenerate && sentUserGenerate) || (!userGenerate && store.hasInterface("update"))) {
-          const isUpdate = sentUserGenerate || versions.result;
+          const isUpdate = isFF(FF_REVIEWER_FLOW) || sentUserGenerate || versions.result;
+          const noChanges = isFF(FF_REVIEWER_FLOW) && !history.canUndo;
+          const isUpdateDisabled = isDisabled || noChanges;
           const button = (
-            <ButtonTooltip key="update" title="Update this task: [ Alt+Enter ]">
+            <ButtonTooltip key="update" title={noChanges ? "No changes were made" : "Update this task: [ Alt+Enter ]"}>
```

Comment:
```
            <ButtonTooltip key="update" title={noChanges ? "No changes were made" : "Update this task: [ Alt+Enter ]"}>
```
Do these changes to disable the Button also disable the hotkey submission as well?

Reply:
With `noChanges` button effectively be disabled, so I think that's okay to not show hotkey at this moment, because it will be useless